app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -13,7 +14,7 @@ jest.mock('@/store/dataStore', () => ({
     NFLTeams: [
       {
         teamId: '1',
-        teamLogo: 'team-a-logo.png',
+        teamLogo: '/team-a-logo.png',
         teamName: 'Packers',
```

Comment:
Hey Chris! What does the `/` in front of 'team-a-logo' do?

Reply:
@vmaineng when using local images it needs a / to route to the public directory.